bootstrap/bootstrap.py

```diff
@@ -90,7 +93,47 @@ def validate_host():
             print("For local development, see http://tljh.jupyter.org/en/latest/contributing/dev-setup.html")
         sys.exit(1)
 
+class LoaderPageRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == "/logs":
+            with open("/opt/tljh/installer.log", "rb") as log_file:
+                content = log_file.read()
+                self.wfile.write(content)
+        elif self.path == "/index.html" or self.path == "/favicon.ico":
+            return SimpleHTTPRequestHandler.do_GET(self)
+        elif self.path == "/":
+            self.send_response(302)
+            self.send_header('Location','/index.html')
+            self.end_headers()
+        else:
+            SimpleHTTPRequestHandler.send_error(self, code=403)
+
+def serve_forever(server):
+    try:
+        server.serve_forever()
+    except KeyboardInterrupt:
+        pass
+
 def main():
+    temp_page_flag = "--temporary-page"
+    if temp_page_flag in sys.argv:
+        # Serve the loading page until TLJH builds
+        index_url="https://raw.githubusercontent.com/GeorgianaElena/the-littlest-jupyterhub/in-progress-page/bootstrap/index.html"
```

Comment:
Hmm, so this means the contents of `index.html` will evolve independently of bootstrap.py. What do you think about embedding the HTML file directly in bootstrap.py?